server/config/database.yml

```diff
@@ -17,16 +17,16 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  username: <%= ENV["DB_USER"] %>
-  password: <%= ENV["DB_PASSWORD"] %>
-  host: localhost
-  # For details on connection pooling, see Rails configuration guide
-  # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+  username: postgres
+  password: passwd
+  host: localhost
+  port: 5432
 
 development:
   <<: *default
-  database: advert_app
+  database: Adv-app
```

Comment:
it seems like those changes may break ability to run project for other teammates, was it agreed within a team?